src/main/java/pl/javastart/task/Drink.java

```diff
@@ -0,0 +1,11 @@
+package pl.javastart.task;
+
+public class Drink {
+    String name;
+    double price;
+    boolean alcoholUsage;
+    String component1;
+    String component2;
+    String component3;
+    int capacity;
```

Comment:
To pole lepiej usunąć, a pojemność drinka wylicz w metodzie main, już po utworzeniu drinka